### ISSUE TITLE
Multi-line import statements creates invalid Apps Script

### ISSUE THREAD

**Issue by byerun** [grant/ts2gas#29]
## Expected Behavior
Multi-line typescript import statements translated correctly

## Actual Behavior
The translated Apps Script code is invalid

## Steps to Reproduce the Problem

1. Create a multi-line typscript import statement such as
~~~~
import {
  FinancialTransactionFromSpreadsheet,
  SpreadSheetDef
} from "./financial.transaction.from.spreadsheet";
~~~~
2. Have clasp push to remote project

The translated code will have a "\n" embedded along with invalid lines:
~~~~ 
//import {
\n  FinancialTransactionFromSpreadsheet,
  SpreadSheetDef
} from "./financial.transaction.from.spreadsheet";
~~~~ 

Attempting to run the code will produce the error message
Illegal character.

## Specifications

- Node version (`node -v`): 11.8.0
- Version (`clasp -v`): 2.0.1
- OS (Mac/Linux/Windows): Windows

**Comment by takanakahiko** [comment]
I think that this line is causing problems.

https://github.com/grant/ts2gas/blob/4f7827164187c6d8deb2c9a259b1312061f2ab0b/src/index.ts#L20

In general, `replace( 'A', 'B' )` replaces A with B only once.
Therefore, if an import statement exceeds two lines, you can not delete a line break completely.

See also : https://www.w3schools.com/jsref/jsref_replace.asp

So, I propose using regular expressions like `replace( /\n/g , '\\n' )`.

**Comment by grant** [comment]
Thanks for the report and fix.
#30 was published with `1.6.1`.

Please create a PR to update this dependency in `clasp`.

**Comment by ardent42c** [comment]
This is still happening with ts2gas 3.6.1 and clasp 2.3.0 ! was the fix rolled back??
